src/java.base/share/classes/jdk/internal/util/HexDigits.java

```diff
@@ -114,24 +114,18 @@ public static short digitPair(int i, boolean ucase) {
     }
 
     /**
-     * Return a little-endian packed integer for the 4 ASCII bytes for an input unsigned 2-byte integer.
-     * {@code b0} is the most significant byte and {@code b1} is the least significant byte.
-     * The integer is passed byte-wise to allow reordering of execution.
+     * Insert the int into the buffer as 4 hexadecimal digits
+     * @param buffer byte buffer to copy into
+     * @param off insert point
+     * @param value to convert
```

Comment:
The parameter is called i. Maybe also add "Only least significant 16 bits are used." Even when it's only a internal api.